pulp_2_tests/tests/puppet/api_v2/test_sync_publish.py

```diff
@@ -57,6 +60,14 @@ def setUpClass(cls):
             cls.repos.append(repo)
             cls.importers_iter.append(client.get(repo['_href'] + 'importers/'))
 
+    @classmethod
+    def tearDownClass(cls):
+        """Delete all resources named by ``resources``."""
+        client = api.Client(cls.cfg)
+        for resource in cls.resources:
+            client.delete(resource)
+        client.delete(ORPHANS_PATH)
```

Comment:
Same suggestion as above.

Reply:
@kersommoura I believe I get it. Please check 54 - 67 on the update. Thanks.